pages/index.tsx

```diff
@@ -367,6 +367,52 @@ const Index = () => {
             </button>
           </div>
         </div>
+        <div
+          className="container"
+          style={{
+            width: "100%",
+            marginBottom: "-32px",
+            padding: "20px 50px",
+            backgroundColor: "var(--light-orange-color)",
+          }}
+        >
+          <h1
+            style={{
+              lineHeight: 1.2,
+              textAlign: "center",
+            }}
+          >
+            Start discovering your career interests now!
+          </h1>
+          <p
+            style={{
+              maxWidth: "1080px",
+              margin: "20px auto",
+              textAlign: "justify",
+              textJustify: "inter-word",
+            }}
+          >
+            Check out our mentors with impressive industry experience on the
+            mentor page and apply today to discover your career interests. Have
+            any questions? Head to our FAQ page to get your burning questions
+            answered!
+          </p>
+          <button
+            className="apply-button"
+            style={{
+              margin: "20px auto",
+              display: "block",
+            }}
+          >
+            <a
+              target="_blank"
+              rel="noopener noreferrer"
+              href="https://advisorysg.typeform.com/to/NQaJmE6j#source=mentorsite"
+            >
+              <h1 style={{ color: "black" }}>Apply Now</h1>
```

Comment:
Although the font size should be big, it isn't ideal to set this as a `h1` since that is semantically intended for text headings. Perhaps you can use Tailwind styles to increase the font size?

Reply:
Made the changes.